embulk-core/src/main/resources/embulk/parent_first_resources.properties

```diff
@@ -1,22 +1,5 @@
 ch/qos/logback/classic/boolex
 ch/qos/logback/classic/db/script
-com/martiansoftware/nailgun/builtins
 embulk
-jni/Darwin
-jni/arm-Linux
-jni/i386-Linux
-jni/i386-SunOS
-jni/i386-Windows
-jni/ppc-AIX
-jni/sparcv9-SunOS
-jni/x86_64-FreeBSD
-jni/x86_64-Linux
-jni/x86_64-SunOS
-jni/x86_64-Windows
-jruby
 msgpack
-net/jcip/annotations
-okay
-org/apache/bval/jsr303
-tables
-templates
+org/embulk
```

Comment:
Is it for those new embulk-util-* resources?

Reply:
Ah, actually not. None of `embulk-util-*` are loaded in the top-level class loader, then they would never conflict.

This is "just in case". I might not need to add this here, but `embulk-core` has some resources in `org/embulk`.
https://github.com/embulk/embulk/tree/v0.10.31/embulk-core/src/main/resources